Jenkinsfile

```diff
@@ -40,9 +40,9 @@ pipeline {
                     steps {
                         sh 'cargo update'
                         // Perform actual check
-                        sh 'cargo check --all --all-targets --features "vulkan sdl_controller json saveload tiles"'
+                        sh 'cargo check --all --all-targets --features "vulkan sdl_controller json saveload tiles storage-event-control"'
```

Comment:
Are those all the features amethyst has? We can also just do --all-features to enable all feature flags.

Reply:
we can't unfortunately -- because of `"vulkan"` / `"empty"` / `"metal"` :art: